test/hard_swish_test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import {hardSwish} from '../src/hard_swish.js';
+import {Tensor} from '../src/lib/tensor.js';
+import * as utils from './utils.js';
+
+describe('test hardSwish', function() {
+  function testHardSwish(input, expected) {
+    const inputTensor = new Tensor(input.shape, input.value);
+    const outputTensor = hardSwish(inputTensor);
+    utils.checkValue(outputTensor, expected);
```

Comment:
Where's the 4 ULP value? (https://github.com/webmachinelearning/webnn/issues/338#issuecomment-1419652594) It appears `checkValue(tensor, expected, nulp = 0)` defaults to zero in https://github.com/webmachinelearning/webnn-baseline/blob/main/test/utils.js.

Reply:
> Where's the 4 ULP value?

This result data computed by pure JavaScript is for baseline data, so it's `0 ULP` here . 
And `4 ULP` is the tolerance for the results computed by WebNN API comparing with baseline data, this `4 ULP` is used by WPT `hardSwish` tests.

Reply:
Okie dokie 👍.